src/features/auth/pages/signup/SignupConfirmationEmailSent/EmailResendModal.native.test.tsx

```diff
@@ -68,17 +68,20 @@ describe('<EmailResendModal />', () => {
 
   it('should display timer when resend email button is clicked', async () => {
     renderEmailResendModal({})
+
     await waitFor(() => {
       expect(screen.getByText('Demander un nouveau lien')).toBeEnabled()
     })
 
-    await act(async () => fireEvent.press(screen.getByText('Demander un nouveau lien')))
+    fireEvent.press(screen.getByText('Demander un nouveau lien'))
 
-    expect(
-      screen.getByText(
-        'Nous t’avons envoyé un nouveau lien. Une autre demande sera possible dans 60s.'
-      )
-    ).toBeOnTheScreen()
+    await waitFor(() => {
+      expect(
+        screen.getByText(
+          'Nous t’avons envoyé un nouveau lien. Une autre demande sera possible dans 60s.'
+        )
+      ).toBeOnTheScreen()
+    })
```

Comment:
je propose de remplacer le waitFor par un find

```suggestion
      expect(
        await screen.findByText(
          'Nous t’avons envoyé un nouveau lien. Une autre demande sera possible dans 60s.'
        )
      ).toBeOnTheScreen()
```

Reply:
ça fonctionne 🥳 merci